docs/README.md

```diff
@@ -27,3 +27,26 @@ If the used Docker release supports multistage builds, you can enable this by se
 ```
 $ MULTISTAGE_BUILD=1 make build
 ```
+
+## Catalog build
+1. Create a new CSV file using the latest ember-csi container. Replace 0.9.x
+   with the next version you want to use.
+```
+$ mkdir -p ../../deploy/olm-catalog/0.9.x
```

Comment:
What directory is this executed from?
Because if it's from the project's root directory, then we are creating a directory outside the repository, and if it's not from there, then the next commands that refer to the `deploy` directory would fail.

Reply:
Fixed